web/src/features/charts/graphUtils.ts

```diff
@@ -211,30 +211,41 @@ export function getElectricityProductionValue({
   return generationTypeStorage === 0 ? 0 : -generationTypeStorage;
 }
 
-function analyzeChartData(chartData: AreaGraphElement[]) {
+function analyzeChartData(chartData: AreaGraphElement[], timeAverage: TimeAverages) {
   let estimatedCount = 0;
   let tsaCount = 0;
+  let allEstimated = true;
+  const isHourly = timeAverage === TimeAverages.HOURLY;
   for (const chartElement of chartData) {
     if (chartElement.meta.estimationMethod === EstimationMethods.TSA) {
       tsaCount++;
     }
```

Comment:
This would need to include the `FORECASTS_HIERARCHY` method as well if we want to ensure it's functioning the same way for Germany and other zones in the future.